### DISCUSSION TITLE
Windows Hello biometric MFA - your account is not validated

### DISCUSSION THREAD

**Discussion by stevealex1** [neos-sdi/adfsmfa#223] [Q&A] [answered]
Hi,

I have setup adfsmfa with my AD FS server hosted in a VM; currently using TOTP and Biometrics as the MFA providers.
I have a fingerprint reader connected to the host machine which is recognised as a Windows Hello fingerprint reader.

From the host machine using the IdP initiated SSO URL, I am trying to login and register the biometric device through a web browser.
After providing the fingerprint at the Windows Hello popup, registration fails with the following message displayed.
_Your account is not validated !
You can try the operation again by selecting "Previous"._

For Windows Hello based authentication over web browser, is it required that the host also be part of the domain?

**Comment by redhook62** [comment]
Hi,

No, it is not required that the machine is linked to the domain (only in case of Windows Hello For Business, but then no need for adfsmfa), for example, you can register your Android or Apple phone or your personal tablet in addition to any PC (Windows, Linux). it is the WebAuthN protocol which is quite open.

I think it's more about the biometrics settings that are not updated according to your needs.

In the Configuration / Security / biometrics, you can test these parameters, they have the advantage of working in the majority of cases with high security options

![image](https://user-images.githubusercontent.com/26826945/159648898-32bfa609-9c3a-4ff7-bd1c-417138716305.png)

In the biometric Provider configuration, don't forget to change the default entries (eg :contoso.com)

![image](https://user-images.githubusercontent.com/26826945/159650165-138c7213-2b33-47df-acfb-5dc9d00d4fb5.png)

Never forget to check the logs in the Event Viewer, there can and should be information that can help you

Let us know.

regards

**Comment by stevealex1** [comment]
Thanks for replying. I have identical configurations for the mentioned parameters.

The device I'm using is fido-UAF certified. Should I be using a fido2/U2F device as a security key?

In Event Viewer, I was not able to find any logs from adfsmfa specifically. AD FS logs did not provide any relevant information.

**Comment by redhook62** [comment, marked as answer]
Hi,

Yes, the fact that your device is compatible with FIDO-UAF and not with FIDO2-U2F is certainly the problem.

However, if you disable extensions (exts) and user verification (uvn), which lowers the level of security, this may pass. 
The list of supported devices is obtained by ADFSMFA from the fido alliance: https://mds.fidoalliance.org.
You can to confirm your tests on 
https://webauthntest.azurewebsites.net/login.html
https://www.passwordless.dev

Regards
